doc/tools/docgram/doc_grammar.ml

```diff
@@ -1753,6 +1755,7 @@ let process_rst g file args seen tac_prods cmd_prods =
   let cmd_replace_files = [
     "doc/sphinx/language/gallina-specification-language.rst";
     "doc/sphinx/language/gallina-extensions.rst"
+    ; "doc/sphinx/proof-engine/vernacular-commands.rst"
```

Comment:
The location of the `;` is inconsistent. Both options (trailing or leading `;`) exist. Choose one and stick to it.

Reply:
I put it there temporarily so I could easily comment out the last element.  Alas, Ocaml doesn't allow a trailing `;`.